CleverPush/CleverPush.xcodeproj/project.pbxproj

```diff
@@ -1530,6 +1530,7 @@
 		294F7E8823FB5DB500806D23 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ARCHS = "$(ARCHS_STANDARD)";
```

Comment:
Can you please check if the changes in this file are all needed for this PR? Also the ones below.